docs/appendix/configuration-file.md

```diff
@@ -1,70 +1,129 @@
+{{% tab name="macOS" %}}
 
-```sh {class="line-numbers linkable-line-numbers"}
-viam-server -config /path/to/config-file.json
-```
+1. Copy the example configuration file to your home directory:
 
-If you are using `systemctl` on Linux to manage `viam-server`, be sure to also update your service file with your custom path.
-You'll need to update these lines in your <file>/etc/systemd/system/viam-server.service</file> file to reflect your custom path:
+   ```sh {class="line-numbers linkable-line-numbers"}
+   cp /opt/homebrew/etc/viam.json ~/viam.json
+   ```
 
-```sh {class="line-numbers linkable-line-numbers"}
-...
-ConditionPathExists=/path/to/config-file.json
-...
-ExecStart=/usr/local/bin/viam-server -config /path/to/config-file.json
-...
-```
+1. Start `viam-server`, providing the custom configuration file location using the `-config` flag:
 
-## Build the configuration file in the Viam app
+   ```sh {class="line-numbers linkable-line-numbers"}
+   viam-server -config ~/viam.json
+   ```
 
-The easiest way to configure your robot and create the configuration file is from the Viam app.
+{{% /tab %}}
+{{% tab name="Linux" %}}
 
-1. Navigate to [the Viam app](https://app.viam.com) and select the **Config** tab.
-1. Use **Builder** mode to add components and services, configure attributes, and map pin assignments.
-1. Then, switch to **Raw JSON** mode to be shown the equivalent JSON configuration to the settings you made in **Builder** mode.
-1. Download the JSON code to the board or computer you intend to run `viam-server` on; click the **See full config** link to be shown this file in plaintext for easy copying.
+1. Copy the example configuration file to your home directory:
 
-If you configure your robot in the **Config** tab before following the steps under the **Setup** tab to install `viam-server`, those steps will download and use the configuration file you've built.
+   ```sh {class="line-numbers linkable-line-numbers"}
+   cp /etc/viam.json ~/viam.json
+   ```
 
-If you later need to make changes to your robot's configuration:
+1. Start `viam-server`, providing the custom configuration file location using the `-config` flag:
 
-1. Make your edits under the **Config** tab in **Builder** mode
-1. Switch to **Raw JSON** mode to view your full configuration in JSON format
-1. Download the code to your robot, overwriting your local configuration file with the new one.
-1. Restart `viam-server` to apply the changes.
+   ```sh {class="line-numbers linkable-line-numbers"}
+   viam-server -config ~/viam.json
+   ```
+
+1. If you are using `systemctl` to manage `viam-server`, you must also update your service file with this custom path.
+   Update these lines in your <file>/etc/systemd/system/viam-server.service</file> file to reflect your custom path:
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   ...
+   ConditionPathExists=~/viam.json
+   ...
+   ExecStart=/usr/local/bin/viam-server -config ~/viam.json

```

Comment:
These won't work. ~ is "my home directory" and will be different for the service running in the background than for a user normally logged in. If they want to run it as a background service, they should actually just use /etc/viam.json as the config file location and not edit the systemd file at all. The second reason for this is that this file may be overwritten during updates. You'll need to note that said file is owned by root, so they'll need sudo to copy to it or edit it.

Reply:
The same goes for the brew service. If (and only if) they want to run it as a background service would they use the default file locations. I don't think you're documenting brew services though (per our discussion, so that's fine) but mention it just for clarification to you.

Reply:
Oh! Yes, you're correct! Thanks - I'll fix this. 👍